test/unit/view.js

```diff
@@ -601,6 +601,186 @@ describe('View class', () => {
+          getComputedStyleStub.restore();
+        });
+
+        it('returns the height of the wrapper set in style', () => {
+          view.wrapper = document.createElement('div');
+          const getComputedStyleStub = sinon.stub(window, 'getComputedStyle').returns(view.wrapper.style);
+          view.wrapper.style.height = '50px';
+          assert.equal(view.outerHeight, '50px');
+          getComputedStyleStub.restore();
+        });
+
+        it('returns wrapper\'s client height if height is not set in style', () => {
+          view.wrapper = {
+            clientHeight: 20,
+            style: {
+              height: '',
+              getPropertyValue: sinon.spy(),
+            },
+          };
+          const getComputedStyleStub = sinon.stub(window, 'getComputedStyle').returns(view.wrapper.style);
+          assert.equal(view.outerHeight, '20px');
+          getComputedStyleStub.restore();
+        });
+      });
+
+      describe('className', () => {
+        it('returns an empty string', () => {
+          assert.equal(view.className, '');
+        });
+      });
+
+      describe('shouldResizeX', () => {
+        it('returns false', () => {
+          assert.equal(view.shouldResizeX, false);
+        });
+      });
+
+      describe('shouldResizeY', () => {
+        it('returns false', () => {
+          assert.equal(view.shouldResizeY, false);
+        });
+      });
+
+      describe('document', () => {
+        it('returns iframe\'s document if iframe exists', () => {
+          view.iframe = {document: {}};
+          assert.equal(view.document, view.iframe.document);
+        });
+
+        it('returns window\'s document if iframe does not exist', () => {
+          view.iframe = null;
+          assert.equal(view.document, window.document);
+        });
+      });
+    });
+
+    describe('"private" methods', () => {
+      describe('_createWrapper()', () => {
+        let createElementStub;
+        let appendStub;
+        let mockWrapper;
+
+        beforeEach(() => {
+          mockWrapper = document.createElement('div');
+          createElementStub = sinon.stub(document, 'createElement').returns(mockWrapper);
+          appendStub = sinon.stub(view, 'append');
+        });
+
+        afterEach(() => {
+          createElementStub.restore();
+          appendStub.restore();
+        });
+
+        it('creates a wrapper and appends it to view', () => {
+          view._createWrapper();
+          assert.calledOnce(createElementStub);
+          assert.calledWith(createElementStub, 'div');
+          assert.calledOnce(appendStub);
+          assert.calledWith(appendStub, mockWrapper);
+        });
+
+        it('returns the newly created wrapper', () => {
+          assert.equal(view._createWrapper(), mockWrapper);
+        });
+
+        it('adds component type class to wrapper', () => {
+          assert.equal(view._createWrapper().className, 'shopify-buy__product');
```

Comment:
Can we identify where this class name is coming from? It seems a bit arbitrary...